CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,5 @@
+# Code of conduct
```

Comment:
Does the file have the be named `CODE_OF_CONDUCT` (with underscores) for Github to recognize it? Will this file then show up for issues and prs?

https://help.github.com/articles/adding-a-code-of-conduct-to-your-project/

Reply:
Hm, I just did what https://raw.githubusercontent.com/mapbox/mapbox-gl-native/master/CODE-OF-CONDUCT.md did